docs/resources/how-it-works.md

```diff
@@ -67,7 +67,7 @@ Currently, the Lit network is in [Mainnet Beta](../network/migration-guide.md) a
 
 ### Node Operators
 
-The current node operators active on the Habanero Mainnet Beta include:
+The current node operators active on the Datil-prod Mainnet Beta include:
```

Comment:
I think we can just say 'Datil Mainnet Beta' here. The node operator list will need to be updated as well, but still uncertain what it will be at launch (based on where folks are at in the setup process)